lib/client/api.go

```diff
@@ -495,6 +495,14 @@ type Config struct {
 
 	// SSHDialTimeout is the timeout value that should be used for SSH connections.
 	SSHDialTimeout time.Duration
+
```

Comment:
We got a conflict here due to some lines that only exist on master. An easy fix.